libnetwork/netavark/config.go

```diff
@@ -202,6 +213,10 @@ func createMacvlan(network *types.Network) error {
 	return nil
 }
 
+func createMockWireGuardNetwork(network *types.Network) error {
+	return nil
+}
```

Comment:
There is no point in having this function then, just drop it.

Reply:
updated